WordPress/src/main/java/org/wordpress/android/ui/bloggingprompts/promptslist/compose/BloggingPromptsListItem.kt

```diff
@@ -106,6 +109,6 @@ fun BloggingPromptsListItemPreview(
     @PreviewParameter(BloggingPromptsListItemPreviewProvider::class) model: BloggingPromptsListItemModel
 ) {
     AppTheme {
-        BloggingPromptsListItem(model)
+        BloggingPromptsListItem(model, Modifier) {}
```

Comment:
Personal preference (optional): to avoid passing the `Modifier` I think it would be good to add the named argument here.


```suggestion
        BloggingPromptsListItem(model, onClick = {})
```